rebasehelper/tests/conftest.py

```diff
@@ -59,3 +61,15 @@ def pytest_collection_modifyitems(items):
             item.add_marker(pytest.mark.functional)
         else:
             item.add_marker(pytest.mark.standard)
+
+
+@pytest.fixture
+def mock_spec_object_generator(monkeypatch):
+    def _mock_spec(content):
```

Comment:
Looking at this, perhaps I could make the fixture a bit more general. There could be an argument for passing methods to monkeypatch and an argument for passing attributes to set. This would make the fixture usable for a lot of test cases. What do you think @nforro ?

Reply:
Sure, but maybe we should consider using something like [flexmock](https://flexmock.readthedocs.io/en/latest/).